### PR TITLE
Add org.fcitx.Fcitx5.Addon.Chewing

### PR DESCRIPTION
**Pull request by wengxt**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by wengxt** [comment]
bot, build org.fcitx.Fcitx5.Addon.Chewing

File: org.fcitx.Fcitx5.Addon.Chewing.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  - /lib/pkgconfig
+build-options:
+  prefix: /app/addons/Chewing
+  prepend-pkg-config-path: /app/addons/Chewing/lib/pkgconfig
+modules:
+  - name: libchewing
+    sources:
+      - type: git
+        url: https://github.com/chewing/libchewing
+        branch: v0.5.1
```
Comment (hfiguiere) [inline org.fcitx.Fcitx5.Addon.Chewing.yaml:21]: you need a tag or commit. not a branch.

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by nedrichards** [comment]
bot, build org.fcitx.Fcitx5.Addon.Chewing

**Comment by nedrichards** [comment]
Do you need the same collaborators on this PR (and https://github.com/flathub/flathub/pull/3310) as you have on the existing fctix extensions and system?

**Pull request closed without merging**